ovn-tester/ovn_workload.py

```diff
@@ -649,12 +649,14 @@ async def create_cluster_join_switch(self, sw_name):
         )
 
     async def provision_ports(self, n_ports, passive=False):
-        ret_list = []
+        tasks = []
+        ctx = ovn_context.active_context
```

Comment:
Hi, 
i think here we can save few seconds by looping over workers instead of ports and create a bundle of ports per ports for each loop iteration i'm not sure if that could really improve the timing of the tests but i saw some improvement on my tests:
```

@@ -651,10 +659,15 @@ class Cluster(object):
     async def provision_ports(self, n_ports, passive=False):
         tasks = []
         ctx = ovn_context.active_context
-        for _ in range(n_ports):
+        
+        for _ in range(len(self.worker_nodes)):
             worker = self.select_worker_for_port()
-            tasks.append(ctx.create_task(worker.provision_ports(self, 1,
+            tasks.append(ctx.create_task(worker.provision_ports(self, (n_ports//len(self.worker_nodes)),
                                                                 passive)))
+        for _ in range(n_ports%len(self.worker_nodes)):
+            worker = self.select_worker_for_port()
+            tasks.append(ctx.create_task(worker.provision_ports(self, 1, passive)))
+
         results = await asyncio.gather(*tasks)
         return [ports[0] for ports in results]
```